opensearch/README.md

```diff
@@ -0,0 +1,112 @@
+# About
+
+This image manages OpenSearch snapshots using the cluster’s native snapshot repository. It simply triggers the REST APIs, so OpenSearch itself handles the data transfer to S3 (or any other repository backend) and keeps the incremental relationships between snapshots.
+
+# Usage
+
+The image contains three scripts that are meant to be run separately, following the same convention as the other database backup images in this repository.
+
+Before running the scripts you must have:
+
+- An OpenSearch cluster reachable from the container.
+- A configured snapshot repository (see the [OpenSearch documentation](https://docs.opensearch.org/latest/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/)).
+- Appropriate TLS material or credentials (client certificate/key, HTTP basic auth, etc.).
+
+Unless mentioned otherwise, all scripts accept the following environment variables:
+
+- `OPENSEARCH_ENDPOINT` (required): Base URL of the OpenSearch cluster, e.g. `https://opensearch:9200`.
+- `OPENSEARCH_SNAPSHOT_REPOSITORY` (required): Name of the repository configured in OpenSearch.
+- `OPENSEARCH_CLIENT_CERT` / `OPENSEARCH_CLIENT_KEY`: Paths to the client certificate and key if certificate-based auth is used.
+- `OPENSEARCH_CA_CERT`: CA certificate used to validate the endpoint. Defaults to the system store unless `OPENSEARCH_SSL_SKIP_VERIFY` is set to `true`.
+- `OPENSEARCH_SSL_SKIP_VERIFY`: Set to `true` to skip TLS verification (not recommended).
+- `OPENSEARCH_USERNAME` / `OPENSEARCH_PASSWORD`: Basic-auth credentials.
+- `OPENSEARCH_WAIT_FOR_COMPLETION`: Whether the API calls should block until completion (`true` by default).
+
+> **Note**: All boolean flags (`..._WAIT_FOR_COMPLETION`, `OPENSEARCH_PRUNE_DRY_RUN`, etc.) only accept the strings `true` or `false` (case-insensitive). Any other value will cause the scripts to exit with an error.
+
+## /opt/backup.py
```

Comment:
Thinking about it, I think I'd use either snapshot or backup everywhere (at least for the opensearch commands) in the terminology to be consistent.

Edit: I think we can just use snapshot everywhere here.